hw/ip/clkmgr/doc/checklist.md

```diff
@@ -0,0 +1,232 @@
+---
+title: "CLKMGR Checklist"
+---
+
+<!--
+NOTE: This is a template checklist document that is required to be copied over to the 'doc'
+directory for a new design that transitions from L0 (Specification) to L1 (Development)
+stage, and updated as needed. Once done, please remove this comment before checking it in.
+-->
+This checklist is for [Hardware Stage]({{< relref "/doc/project/development_stages.md" >}}) transitions for the [CLKMGR peripheral.]({{< relref "hw/ip/clkmgr/doc" >}})
+All checklist items refer to the content in the [Checklist.]({{< relref "/doc/project/checklist.md" >}})
+
+## Design Checklist
+
+### D1
+
+Type          | Item                           | Resolution  | Note/Collaterals
+--------------|--------------------------------|-------------|------------------
+Documentation | [SPEC_COMPLETE][]              | WIP         | [CLKMGR Design Spec]({{<relref "hw/ip/clkmgr/doc" >}})
```

Comment:
can we consider this done instead of WIP? updates to the spec are always possible in D2...

Reply:
fixed.